config/services.yaml

```diff
@@ -14,6 +14,7 @@ parameters:
   catrobat.file.storage.dir:            "%kernel.project_dir%/public/resources/programs/"
   catrobat.file.storage.path:           "resources/programs/"
   catrobat.logs.dir:                    "%kernel.project_dir%/var/log/"
+  catrobat.max_name_upload_size: 1000
```

Comment:
Have you checked the app implementation? 1000 seems a lot for the project name.

@Tamdav Do you have any knowledge about this on the android side?
Or any UX recommendations? :)

Reply:
There are no restrictions for this on the android side. However, I would suggest a maximum of 50 characters, which should be quite enough for the project name.